english/marshmallow.md

```diff
@@ -1,3 +1,4 @@
+If you like [badminton](badminton/bad.md).
 [Want to see my Quora profile](ismail-quora-profile/quora-profile.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.